### PR TITLE
add securityProfile to create parameters

### PR DESCRIPTION
**Pull request by giyer**
This changesets adds a new securityProfile section that will allow
integration of a HDInsight cluster with active directory.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@giyer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (giyer). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by amarzavery** [comment]
@giyer - Can you update the package.json version over [here](https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/HDInsight2/package.json)

File: lib/services/HDInsight2/lib/hDInsightManagementClient.js [hunk 1]
```diff
@@ -276,7 +276,7 @@ var ClusterOperations = ( /** @lends ClusterOperations */ function() {
     
     // Set Headers
     httpRequest.headers['Content-Type'] = 'application/json';
-    httpRequest.headers['User-Agent'] = 'ARM SDK Node v0.2.0';
+    httpRequest.headers['User-Agent'] = 'ARM SDK Node v0.2.1';
```
Comment (amarzavery) [inline lib/services/HDInsight2/lib/hDInsightManagementClient.js]: once you update the version in package.json. Please make sure that the same version is reflected over here.

**Comment by giyer** [comment]
@amarzavery - Thanks for taking a look! I submitted an update after changing version to 0.2.2

**Comment by amarzavery** [comment]
@giyer - The PR has been merged. Please take a look at [this link](https://github.com/Azure/azure-sdk-for-node/blob/master/Documentation/RegeneratingCode.md#publishing-to-npm) that provides instructions on how to publish your package to npm.

**Pull request merged**